### PR TITLE
fix: show guide in the firmware modal, refactor modal header

### PR DESCRIPTION
**Pull request by dahaca**
fix the second bug https://github.com/trezor/trezor-suite/issues/4739#issuecomment-1095153000

### CODE REVIEW

**Review by tomasklim — CHANGES_REQUESTED** [review]
Screenshot 2022-04-14 at 9 34 08

File: packages/suite/src/views/firmware/index.tsx [hunk 1]
```diff
@@ -12,27 +12,21 @@ import { DeviceAcquire, DeviceUnknown, DeviceUnreadable } from '@suite-views';
 import { Translation, Modal } from '@suite-components';
 import { OnboardingStepBox } from '@onboarding-components';
 import { useActions, useFirmware, useSelector } from '@suite-hooks';
-import { ConfirmOnDevice, Icon, useTheme } from '@trezor/components';
+import { ConfirmOnDevice, variables } from '@trezor/components';
 import * as suiteActions from '@suite-actions/suiteActions';
 
-const Wrapper = styled.div`
+const Wrapper = styled.div<{ withTopPadding: boolean }>`
```
Comment (Nodonisko) [inline packages/suite/src/views/firmware/index.tsx]: I would prefer `hasTopPadding` or `isWithTopPadding`

Comment (dahaca) [reply packages/suite/src/views/firmware/index.tsx]: done

**Review by tomasklim — APPROVED** [review]

**Comment by STew790** [comment]
**QA OK**

Guide is shown in the firmware modal.

image

**Info:**
 - Suite version: web 22.5.0 (bf0e4020dc1344c179501fcb5f57d9c4039c8bf0)
 - Browser: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.4896.127 Safari/537.36
 - OS: Linux x86_64
 - Screen: 1920x1080
 - Device: 
 - Transport: bridge 2.0.31

**Pull request merged**